seccomp/mount.go

```diff
@@ -181,8 +189,10 @@ func (m *mountSyscallInfo) createProcPayload(
 	// apply this to the new mountpoint (otherwise we will get a permission
 	// denied from the kernel when doing the mount).
 	procInfo := mip.GetInfo("/proc")
-	if _, ok := procInfo.VfsOptions["ro"]; ok == true {
-		payload[0].Flags |= unix.MS_RDONLY
+	if procInfo != nil {
```

Comment:
When is procInfo null? And what do we do when it is null?

Reply:
This is the fix for issue [#82](https://github.com/nestybox/sysbox/issues/82). Basically, we were assuming that a procfs is always present, which is not necessarily the case in chroot jails.

If there's no procfs (procinfo == null) we just keep on going w/o dereferencing the null pointer.